src/axom/spin/BVH.hpp

```diff
@@ -336,23 +314,13 @@ class BVH
    * \pre offsets    != nullptr
    * \pre counts     != nullptr
    * \pre candidates == nullptr
-   * \pre xmin != nullptr
-   * \pre xmax != nullptr
-   * \pre ymin != nullptr
-   * \pre ymax != nullptr
-   * \pre zmin != nullptr if dimension==3
-   * \pre zmax != nullptr if dimension==3
+   * \pre boxes != nullptr
    */
   void findBoundingBoxes(IndexType* offsets,
```

Comment:
This, and the other new find() functions are much less error-prone for the caller than the original versions.